shared/lib_irradproc.cpp

```diff
@@ -1134,11 +1162,9 @@ int irrad::calc()
 	else
 	{	
 		// sun is down, assign sundown values
-		sunAnglesRadians[0] = -999*DTOR; //avoid returning a junk azimuth angle (return in radians)
-		sunAnglesRadians[1] = -999*DTOR; //avoid returning a junk zenith angle (return in radians)
-		sunAnglesRadians[2] = -999*DTOR; //avoid returning a junk elevation angle (return in radians)
-		timeStepSunPosition[0] = 0;
-		timeStepSunPosition[1] = 0;
+		solarpos(year, month, day, hour, minute, latitudeDegrees, longitudeDegrees, timezone, sunAnglesRadians);
```

Comment:
Is this calculation necessary since timeStepSunPosition[2] = 0; and we will go straight to line 1237?

Reply:
Yes, this calculation is necessary because timeStepSunPosition gets reported as a result, so it must be assigned using the right hour or they will report the results from noon (which is why previously they were overwritten with -999 and 0).